packages/EasyErrorHandler/src/Bridge/Laravel/Translator.php

```diff
@@ -24,8 +25,24 @@ public function __construct(IlluminateTranslatorInterface $decorated)
      */
     public function trans(string $message, array $parameters): string
     {
-        // TODO: rework after upgrading all the illuminate and laravel packages to ^6.0
-        $method = \method_exists($this->decorated, 'trans') ? 'trans' : 'get';
+        $translation = $this->getOrLang($message, $parameters);
+
+        if ($translation !== $message) {
+            return $translation;
+        }
+
+        return $this->getOrLang(
```

Comment:
What happen if the translation doesn't exist even in the package namespace?

Reply:
Like as usual, it will return as it is, however with a namespace on it. Like:
```json
{
  "message": "easy-error-handler:non-existing-error",
  "subcode": 0,
  ...
}
```

Reply:
@jayaregalinadaflexisourceit ok do you think that's a good solution? 😄

Reply:
hmmm, according to the test, I think nope, I think I should the `startWith` checker here.

Reply:
@jayaregalinadaflexisourceit ok I agree it's not a good solution haha

But no need for `startWith`, just assign the message prefixed with the namespace to a variable, proceed with translation and if the result is the same as the variable you created then you return the original message 😉

Reply:
Ok, I think that is even better, and less dependent. I'll try that now